### PR TITLE
chore(package): update momentum-ui/react version to fix the multiselect of choiceset

### PR DESCRIPTION
**Pull request by mokandre**
Description of changes made (Required)
* Upgraded the momentum UI react version 
* Updated the snapshots

Associated issue number/tracking link (Required)
[SPARK-136312](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-136312)

**The following criteria should be met (Required)**
- [X] [commit guidelines](https://sqbu-github.cisco.com/WebExSquared/web-client/blob/master/CONTRIBUTING.md#github-commit-guidelines) were followed
- [ ] Tests have been added for new functionality and all existing tests pass.
- [X] [pushing guidelines](https://sqbu-github.cisco.com/WebExSquared/web-client/blob/master/CONTRIBUTING.md#pushing-to-the-pipeline) are followed

Reviewers (Required)
@adamweeks

### CODE REVIEW

File: packages/node_modules/@webex/widget-recents/src/components/__snapshots__/RecentsHeader.test.js.snap [hunk 1]
```diff
@@ -18,7 +18,7 @@ exports[`RecentsHeader component renders correctly 1`] = `
     <div
       className="webex-search-input searchInput"
     >
-      <SearchInput
+      <UNDEFINED
```
Comment (lalli-flores) [inline packages/node_modules/@webex/widget-recents/src/components/__snapshots__/RecentsHeader.test.js.snap:21]: This change doesn't look right, will this break something in the widgets?

**Comment by lalli-flores** [comment]
@mokandre the snapshot for `RecentsHeader` shows the component as `UNDEFINED`, have you verified that the widgets are not breaking by this update?

**Comment by mokandre** [comment]
> @mokandre the snapshot for `RecentsHeader` shows the component as `UNDEFINED`, have you verified that the widgets are not breaking by this update?

I just checked about that `RecentsHeader` snapshot. It seems changed to `InputSearch` from `SearchInput`(https://momentum.design/components/input-search/code#default). Can I make that change?

File: packages/node_modules/@webex/widget-recents/src/components/RecentsHeader.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-import {Button, Icon, SearchInput} from '@momentum-ui/react';
+import {Button, Icon, InputSearch} from '@momentum-ui/react';
```
Comment (mokandre) [inline packages/node_modules/@webex/widget-recents/src/components/RecentsHeader.js:5]: It got changed from `SearchInput` to `InputSearch` (https://momentum.design/components/input-search/code#default) 

Working recent widgets after the change:

Screenshot 2020-07-09 at 4 41 10 AM

Comment (lalli-flores) [reply packages/node_modules/@webex/widget-recents/src/components/RecentsHeader.js:5]: Thanks for addressing the breaking change

**Comment by lalli-flores** [comment]
> Can I make that change in `RecentsHeader`

Of course, we don't want to introduce code that breaks other stuff 🙂

**Comment by lalli-flores** [comment]
Seems like there are failing unit tests still, see the [failed CI job](https://app.circleci.com/pipelines/github/webex/react-widgets/704/workflows/b308dac5-ad33-4177-b579-82876e47a866/jobs/12109)

**Pull request merged**